packages/common/src/api/tan-query/utils/parsePurchase.ts

```diff
@@ -0,0 +1,33 @@
+import { full } from '@audius/sdk'
+
+import { PurchaseAccess } from '~/models'
+import {
+  USDCContentPurchaseType,
+  USDCPurchaseDetails
+} from '~/models/USDCTransactions'
+import { StringUSDC } from '~/models/Wallet'
+
+import { HashId } from '../../utils'
+
+export const parsePurchase = (purchase: full.Purchase): USDCPurchaseDetails => {
```

Comment:
This probably could be rewritten as an sdk adapter and put in the adapters folder, fwiw.
I know it's existing code, though, so okay to ignore for now if you'd like.